sdk/core/azure-core/inc/azure/core/internal/tracing/service_tracing.hpp

```diff
@@ -2,9 +2,9 @@
 // Licensed under the MIT License.
 
 #include "azure/core/context.hpp"
+#include "azure/core/http/http.hpp"
```

Comment:
You probably noticed this, so I am curious, what type/usage is requiring adding this header to **this** file, after we remove the user agent header?

I don't see usage of the useragent here. And that header didn't include anything but `<string>`, so I am surprised this is needing it.

If you remove this header, do you get a build error?

Reply:
Thank you, I think this is a good catch, I dropped that include. Previous include shouldn't have been there as well.